gigascience/gigadb-website
# Feat 2117 rewritting the api endpoint returning dataset as xml

# Pull request for issue: #2117 

This is a pull request for the following functionalities:

will need https://github.com/gigascience/gigadb-website/pull/1978 and  https://github.com/gigascience/gigadb-website/pull/2118 merged

Rewriting the API endpoint that returns the Dataset in XML format

No need for the four different templates.
Everything could be handled via a single Yii2 service.
The parameters and the way the API is called should remain the same as before.

## How to test?

Given I follow the instructions on the API help page (https://gigadb.org/site/help#interface)
When I select the example as shown "http://gigadb.org/api/dataset?doi=100051"
Then I see the correct results

## How have functionalities been implemented?

## Any issues with implementation?

## Any changes to automated tests?

tests added

## Any changes to documentation?

## Any technical debt repayment?

## Any improvements to CI/CD pipeline?
